src/main/java/com/google/sps/servlets/DataFromDatabaseServlet.java

```diff
@@ -35,6 +40,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.setContentType(constants.TEXT_TYPE);
     selectedTables = request.getParameterValues(constants.TABLE_SELECT_PARAM);
     String databaseName = request.getParameter(constants.DATABASE_PARAM);
+    String reason = request.getParameter("reason");
```

Comment:
I'm not sure if we currently add the reason to the request. Did you make sure of this?

Reply:
Yeah it is added!